src/utils/compose.ts

```diff
@@ -171,25 +170,32 @@ export const createRelease = async function (
 
 	return {
 		client: pinejsClient,
-		release: _.pick(release, [
-			'id',
-			'status',
-			'commit',
-			'composition',
-			'source',
-			'is_final',
-			'contract',
-			'semver',
-			'start_timestamp',
-			'end_timestamp',
-		]),
-		serviceImages: _.mapValues(
-			serviceImages,
-			(serviceImage) =>
-				_.omit(serviceImage, ['created_at', 'is_a_build_of__service']) as Omit<
-					typeof serviceImage,
-					'created_at' | 'is_a_build_of__service'
-				>,
+		release: Object.fromEntries(
+			Object.entries(release).filter(([key]) =>
+				[
+					'id',
+					'status',
+					'commit',
+					'composition',
+					'source',
+					'is_final',
+					'contract',
+					'semver',
+					'start_timestamp',
+					'end_timestamp',
+				].includes(key),
+			),
+		) as typeof release,
+		serviceImages: Object.fromEntries(
+			Object.entries(serviceImages).map(([key, serviceImage]) => {
+				if ('created_at' in serviceImage) {
+					delete serviceImage.created_at;
+				}
+				if ('is_a_build_of__service' in serviceImage) {
+					delete serviceImage.is_a_build_of__service;
+				}
+				return [key, serviceImage];
+			}),
```

Comment:
The new version already mutates the existing entries (which might be a problem, but assuming it's not) so you can simplify this to purely use that mutation via:
```suggestion
	Object.values(serviceImages).map((serviceImage) => {
		if ('created_at' in serviceImage) {
			delete serviceImage.created_at;
		}
		if ('is_a_build_of__service' in serviceImage) {
			delete serviceImage.is_a_build_of__service;
		}
	});

	...

	return {
		...,
		serviceImages,
```
And it might also be possible to drop those `in` checks, but I don't know

Reply:
> The new version already mutates the existing entries (which might be a problem

`serviceImages` is retrieved from a Promise directly above and not used after this, so I think mutating here should be fine, but if you can think of a reason it should not be, please let me know

> And it might also be possible to drop those in checks, but I don't know

Sadly no, I initially tried without them and it errors with `Property 'X' does not exist on type 'ImageModel'.` I tried it again just to be sure and it indeed seems still necessary